doc/DeveloperGuide.md

```diff
@@ -88,6 +88,28 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook requests requests for confirmation
```

Comment:
The use case seeks to model the interaction between an actor and the system.
Therefore the interaction should be alternating between user and the system, `user -> system -> user -> system -> ...`
In this MSS, do you think there's something that could be changed to reflect this?